b_cfn_elasticsearch_cloner/resource.py

```diff
@@ -145,6 +154,9 @@ def __init__(
                 "ES_INDEX_NAME": elasticsearch_index.index_name,
                 "ES_DOMAIN_ENDPOINT": elasticsearch_index.elasticsearch_domain.domain_endpoint,
                 "PRIMARY_KEY_FIELD": primary_key_field,
+                "EMBEDDINGS_KEY": embeddings_key or '',
+                "SAGEMAKER_ENDPOINT_NAME": sagemaker_endpoint_name or '',
+                "SAGEMAKER_EMBEDDINGS_KEY": sagemaker_embeddings_key or ''
```

Comment:
dont pass optional env vars as empty strings! Append filtered (non `None`) vars instead, like in this example:
```python
optional_items = {
  'optC': ...,
  `optD`: None
}
items={
  'A': ...,
  'B': ...,
  **{k: v for k, v in optional_items if v is not None}
}

>>> items
{
  'A': ...,
  'B': ...,
  'optC': ...
}
```